unittests/v1.2.0/test_report_generation.py

```diff
@@ -0,0 +1,79 @@
+from compliance_suite.report_runner import *
+
+auth_types = ["none", "basic", "bearer", "passport"]
+
+def test_report_runner():
+
+    port_numbers = [8089, 8090, 8091, 8092]
+
+    for authtype, port in zip(auth_types, port_numbers):
+        good_report = report_runner(server_base_url = "http://localhost:" + str(port) + "/ga4gh/drs/v1",
+                                    platform_name = "good mock server",
+                                    platform_description = authtype,
+                                    drs_version = "1.2.0",
+                                    config_file = "compliance_suite/config/config_samples/config_" + authtype + ".json")
+
+        good_json = json.loads(good_report.to_json())
+
+        # remove timestamps, otherwise assert will fail 100%
+        good_json["start_time"] = ""
+        good_json["end_time"] = ""
+        for phase in good_json["phases"]:
+            phase["start_time"] = ""
+            phase["end_time"] = ""
+            for test in phase["tests"]:
+                test["start_time"] = ""
+                test["end_time"] = ""
+                for case in test["cases"]:
+                    case["start_time"] = ""
+                    case["end_time"] = ""
+
+        # USE THIS: uncomment below to generate a new expected_[none/basic/bearer/passport].json
+        # with open('unittests/output/expected_' + authtype + '.json', 'w', encoding="utf-8") as f: json.dump(good_json, f)
+
+        expect_final_json = json.loads(
+            open("unittests/output/expected_" + authtype + ".json", "r").read()
+        )
+
+        actual_json_s = str(good_json).replace("'", '"').replace("\\","")
+        expect_json_s = str(expect_final_json).replace("'", '"').replace("\\","")
+
+        # expect_final_json will vary throughout production of cs
```

Comment:
we decided that we would add the code to check statuses from `test_validate_response` here. And also, `git rm test_validate_response.py`